### PR TITLE
Add Icons for email and password fields #1149

### PR DESCRIPTION
**Pull request by SudhanvaMG**
screenshot from 2018-03-19 01-18-47

Demo Link: lean-sheep.surge.sh

### CODE REVIEW

**Comment by SudhanvaMG** [comment]
@Orbiter @duttaditya18 Please review the PR

**Comment by SudhanvaMG** [comment]
@anshumanv this link is provided below the screen shot .

**Comment by SudhanvaMG** [comment]
@duttaditya18 , yes i know . Had a bit of problem in the build so had to close the previous PR thats all .

**Comment by SudhanvaMG** [comment]
@psikarwal  The link is provided below the screenshot , i have mentioned it as demo link . (lean-sheep.surge.sh)

**Comment by psikarwal** [comment]
Oops, I missed that 😅.

File: src/components/Auth/Login/Login.react.js [hunk 1]
```diff
@@ -13,8 +13,15 @@ import UserPreferencesStore from '../../../stores/UserPreferencesStore';
 import CustomServer from '../../ChatApp/CustomServer.react';
 import Translate from '../../Translate/Translate.react';
 import Dialog from 'material-ui/Dialog';
+import CommunicationEmail from 'material-ui/svg-icons/communication/email';
+import ActionLock from 'material-ui/svg-icons/action/lock'
+
 const cookies = new Cookies();
 
+const iconStyles = {
+  marginRight: 10,
```
Comment (daminisatya) [inline src/components/Auth/Login/Login.react.js]: Do you think this will not affect the responsiveness?

Comment (duttaditya18) [reply src/components/Auth/Login/Login.react.js]: It doesn't. Without the `margin-right`, the icon remains on the same line as the input box if the screen size is small. See this : 

With/Without Margin

**Comment by SudhanvaMG** [comment]
@daminisatya Made one commit and i have deleted the margin .

**Comment by daminisatya** [comment]
Cool, please update the PR with the margin and also make sure you align it with the placeholder.

**Comment by SudhanvaMG** [comment]
@daminisatya  yes, i have updated the PR with the margin , it aligns with the place holder now .
icon

File: src/components/Auth/Login/Login.react.js [hunk 2]
```diff
@@ -258,6 +260,7 @@ class Login extends Component {
 					<h3><Translate text="Login to SUSI"/></h3>
 					<form onSubmit={this.handleSubmit}>
 						<div>
+						<CommunicationEmail/>
 							<TextField name="email"
```
Comment (daminisatya) [inline src/components/Auth/Login/Login.react.js]: where in the code the margin is given? im not able to locate the line

Comment (SudhanvaMG) [reply src/components/Auth/Login/Login.react.js]: @daminisatya  There is no margin , it doesn't allow responsiveness . The UI looks much better in default values .
U want me to add margin and change it adopt responsiveness ? or this looks fine ?

**Comment by daminisatya** [comment]
Looks good thank you.

**Pull request merged**